std_execution.bs

```diff
@@ -3194,13 +3194,13 @@ enum class forward_progress_guarantee {
     </pre>
 
 6. If `value_types_of_t<S, Tuple, Variant>` for some sender type `S` is well formed, it shall be a type <code>Variant&lt;Tuple&lt;Args<sub>0</sub>...>, Tuple&lt;Args<sub>1</sub>...>, ..., Tuple&lt;Args<sub>N</sub>...>>></code>, where the type packs <code>Args<sub>0</sub></code> through <code>Args<sub>N</sub></code> are the packs of types the sender `S` passes as
-    arguments to `execution::set_value` after a receiver object. If such sender `S` invokes `execution::set_value(r, args...)` for some receiver `r`, where `decltype(args)` is not one of the type packs <code>Args<sub>0</sub></code> through <code>Args<sub>N</sub></code>, the program is ill-formed with no
+    arguments to `execution::set_value` after a receiver object. If such sender `S` odr-uses ([basic.def.odr]) `execution::set_value(r, args...)` for some receiver `r`, where `decltype(args)...` is not one of the type packs <code>Args<sub>0</sub>...</code> through <code>Args<sub>N</sub>...</code>, the program is ill-formed with no
```

Comment:
FWIW, I think requiring _exact_ type matching here seems over-strict to me. If a sender reports that it sends an `int` but actually sends an `int&` or `const int&`, IFNDR seems a bit harsh. I'm also curious if we really mean `decltype(args)` here instead of `decltype((args))`. Only questions, no answers at this point.

Reply:
Right, but if it reports that it sends `int &` but actually sends an rvalue, that's problematic, right?

Reply:
Agreed. I haven't yet come up with a way of matching types here that says what we want it to say, mostly because I don't know yet exactly what we want to say.

Reply:
I've filed this as #302 so we don't lose it.